torch/testing/_internal/common_methods_invocations.py

```diff
@@ -19189,6 +19189,15 @@ def sample_inputs_alias_copy(op_info, device, dtype, requires_grad, **kwargs):
                DecorateInfo(unittest.skip("Skipped!"), 'TestMathBits', 'test_neg_view'),
                # UserWarning not triggered : Resized a non-empty tensor but did not warn about it.
                DecorateInfo(unittest.expectedFailure, 'TestCommon', 'test_out_warning'),
+               # "where_cpu" not implemented for float8_e4m3fn
+               DecorateInfo(unittest.expectedFailure, 'TestCommonCPU', 'test_python_ref',
+                            dtypes=(torch.float8_e4m3fn, torch.float8_e5m2, torch.float8_e4m3fnuz, torch.float8_e5m2fnuz)),
+               DecorateInfo(unittest.expectedFailure, 'TestCommonCPU', 'test_python_ref_torch_fallback',
+                            dtypes=(torch.float8_e4m3fn, torch.float8_e5m2, torch.float8_e4m3fnuz, torch.float8_e5m2fnuz)),
+               DecorateInfo(unittest.expectedFailure, 'TestCommonCPU', 'test_python_ref_meta',
+                            dtypes=(torch.float8_e4m3fn, torch.float8_e5m2, torch.float8_e4m3fnuz, torch.float8_e5m2fnuz)),
+               DecorateInfo(unittest.expectedFailure, 'TestDecompCPU', 'test_comprehensive',
+                            dtypes=(torch.float8_e4m3fn, torch.float8_e5m2, torch.float8_e4m3fnuz, torch.float8_e5m2fnuz)),
```

Comment:
What's the point of extending the test if it's going to fail anyway?

Reply:
I have added support for where_cpu and where_cuda. These skips are no need any more.